test/unit/collection-view.spec.js

```diff
@@ -990,44 +992,41 @@ describe('collection view', function() {
       this.collectionView = new this.CollectionView({
         collection: this.collection
       });
-      $('body').append(this.collectionView.el);
+      $('#fixture-container').append(this.collectionView.el);
 
       this.collectionView.render();
-      this.collectionView.onShow();
       this.collectionView.trigger('show');
 
       this.sinon.spy(this.collectionView, 'attachBuffer');
       this.sinon.spy(this.collectionView, 'getChildView');
 
       this.collection.add(this.model2);
-      this.view = this.collectionView.children.findByIndex(1);
+      this.childView2 = this.collectionView.children.findByIndex(1);
     });
 
     it('should not use the render buffer', function() {
       expect(this.collectionView.attachBuffer).not.to.have.been.called;
     });
 
-    it('should call the "onBeforeShow" method of the child view', function() {
-      expect(this.ChildView.prototype.onBeforeShow).to.have.been.called;
-    });
-
-    it('should call the childs "onBeforeShow" method with itself as the context', function() {
-      expect(this.ChildView.prototype.onBeforeShow).to.have.been.calledOn(this.view);
+    it('should call onBeforeShow of the added child view', function() {
+      expect(this.childView2.onBeforeShow).to.have.been.called.once;
```

Comment:
Could you change to `to.have.been.calledOnce`? Its what we are using throughout the suite :)